test/route.test.js

```diff
@@ -115,7 +115,7 @@ test('route', t => {
 
   fastify.listen({ port: 0 }, function (err) {
     if (err) t.error(err)
```

Comment:
this `if` around `t.error` can likely be removed following what's done in most other test files.